scss/modules/_data-container.scss

```diff
@@ -76,6 +76,54 @@
   padding-top: u(.5rem);
 }
 
+.report-type-col {
```

Comment:
More of a stylistic thing, but what about renaming to `.column---download` and putting it in `_datatables.scss` where the other `.column---*` classes live?